io.github.thetumultuousunicornofdarkness.cpu-x.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: io.github.thetumultuousunicornofdarkness.cpu-x
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

Comment:
What's the reason to prefer 22.08? This was 23.08

Reply:
Glibc version is 2.38 in SDK 23.08 and I have compatibility issue with this one, that is why I downgraded to 22.08.